go.mod

```diff
@@ -67,3 +67,5 @@ require (
 )
 
 go 1.13
+
+replace github.com/pingcap/parser => github.com/rebelice/parser master
```

Comment:
You don't need to change this (and should not), use `/run-all-tests parser=pr/xxx` to specify a parser PR to test against.